tools/extension-check/README.md

```diff
@@ -0,0 +1,12 @@
+# Extension check
+
+This is a simple tool to check all core extensions from envoy are enabled in [istio/proxy](https://github.com/istio/proxy).
+
+## Usage
+
+```bash
+go run tools/extension-check/main.go \
+  --ignore-extensions "<WELLKNOWN_IGNORE_EXTENSIONS>" \
+  --envoy-extensions-build-config "<ENVOY_EXTENSIONS_BUILD_CONFIG>" \
+  --proxy-extensions-build-config "<PROXY_EXTENSIONS_BUILD_CONFIG>"
+```
```

Comment:
Can you add an example command?

Reply:
there's one in `check-style.sh`

Reply:
The one in check-style.sh is the same as the one above. An example with concrete values for WELLKNOWN_IGNORE_EXTENSIONS, ENVOY_EXTENSIONS_BUILD_CONFIG and PROXY_EXTENSIONS_BUILD_CONFIG will make it easer for an user. An user should not have to look into the code to figure out example values for ENVOY_EXTENSIONS_BUILD_CONFIG and PROXY_EXTENSIONS_BUILD_CONFIG. 

go run tools/extension-check/main.go \
  --ignore-extensions tools/extension-check/wellknown-extensions \
  --envoy-extensions-build-config "${ENVOY_EXTENSIONS_BUILD_CONFIG}" \
  --proxy-extensions-build-config "${PROXY_EXTENSIONS_BUILD_CONFIG}"

Reply:
I don't expect anyone use this out of istio, but I will try to add more examples.
There's a hack for proxy.

Reply:
updated